glidertest/summary_sheet.py

```diff
@@ -0,0 +1,228 @@
+        "location_test": {"bbox": [10, 50, 25, 70]},
+    },
+    "PSAL": {"gross_range_test": {"suspect_span": [5, 38], "fail_span": [2, 41]},
+             "spike_test": {"suspect_threshold": 0.3, "fail_threshold": 0.9},
+             "location_test": {"bbox": [10, 50, 25, 70]},
+             },
+    "DOXY": {"gross_range_test": {
+        "suspect_span": [0, 350],
+        "fail_span": [0, 500],
+    },
+        "spike_test": {"suspect_threshold": 10, "fail_threshold": 50},
+        "location_test": {"bbox": [10, 50, 25, 70]},
+    },
+    "CHLA": {"gross_range_test": {
+        "suspect_span": [0, 15],
+        "fail_span": [-1, 20],
+    },
+        "spike_test": {"suspect_threshold": 1, "fail_threshold": 5},
+        "location_test": {"bbox": [10, 50, 25, 70]},
+    },
+}
+
+
+def qc_checks(ds, var='TEMP'):
+    gr = tools.compute_global_range(ds, var=var, min_val=configs[var]['gross_range_test']['suspect_span'][0],
+                                    max_val=configs[var]['gross_range_test']['suspect_span'][1])
+    spike = qartod.spike_test(ds[var], suspect_threshold=configs[var]['spike_test']['suspect_threshold'],
+                              fail_threshold=configs[var]['spike_test']['fail_threshold'], method="average")
+    flat = qartod.flat_line_test(ds[var], ds.TIME, 1, 3, 0.001)
+    __, __, err, __ = tools.compute_hyst_stat(ds, var=var, v_res=1)
+
+    return gr, spike, flat, err
+
+
+def fill_tableqc(df,ds, var='TEMP'):
+    if var == 'TEMP':
+        variable = 'Temperature'
+    if var == 'PSAL':
+        variable = 'Salinity'
+    if var == 'DOXY':
+        variable = 'Oxygen'
+    if var == 'CHLA':
+        variable = 'Chlorophyll'
+    if var not in ds.variables:
+        df.loc[:, variable] = 'No data'
+    else:
+        gr, spike, flat, err = qc_checks(ds, var=var)
+        if len(gr) > 0:
+            df.loc[0, variable] = 'X'
+        if len(np.where((spike == 3) | (spike == 4))[0]) > 0:
+            df.loc[1, variable] = 'X'
+        if len(np.where((flat == 3) | (flat == 4))[0]) > 0:
+            df.loc[2, variable] = 'X'
+        if len(np.where(err > 5)[0]) > 5:
+            df.loc[3, variable] = 'X'
+    return df
+
+
+def phrase_duration_check(ds):
+    duration = tools.compute_prof_duration(ds)
+    rolling_mean, overtime = tools.find_outlier_duration(duration, rolling=20, std=2)
+    if len(overtime) > 0:
+        duration_check = f'{len(overtime)} profiles have abnormal duration'
+    else:
+        duration_check = 'No issues with profile duration have been detected'
+    return duration_check
+
+
+def phrase_numberprof_check(ds):
+    check = tools.check_monotony(ds.PROFILE_NUMBER)
+    if check:
+        prof_check = 'No issues detected'
+    else:
+        prof_check = 'Issues with profile number have been detected'
+    return prof_check
+
+# Page 1 with general info and location of the mission
+def summary_plot(ds, save_dir='blabla'):
```

Comment:
default arguments should work by default. It is quite unlikely the user has made a directory 'blabla'. How about the default of save_dir='.', which is the current directory the user is working in.